src/foam/doc/DigSelectView.js

```diff
@@ -0,0 +1,134 @@
+foam.CLASS({
+  package: 'foam.doc',
+  name: 'DigSelectView',
+  extends: 'foam.u2.View',
+  implements: [
+    'foam.mlang.Expressions',
+  ],
+  imports: [
+    'appConfig',
+    'user',
+  ],
+  requires: [
+    'foam.doc.CodeTabs',
+    'foam.u2.Tab',
+    'foam.doc.dao.AxiomDAO',
+    'foam.doc.PropertyAxiom',
+    'foam.doc.CodeTab',
+    'foam.flow.PromiseSlot',
+    'foam.u2.DetailView',
+    'foam.nanos.dig.DIG',
+  ],
+  properties: [
+    {
+      class: 'StringArray',
+      name: 'digProperties',
```

Comment:
You don't appear to be using this at all. Remove it from here and remove the reference to it in the slot below.